x-pack/plugin/logsdb/src/yamlRestTest/resources/rest-api-spec/test/40_source_mode_setting.yml

```diff
@@ -590,237 +467,25 @@ modify final setting after index creation:
             mapping.source.mode: synthetic
 
 ---
-modify source mapping from stored to disabled after index creation:
-  - do:
-      indices.create:
-        index: test_modify_source_mode_stored_disabled
-        body:
-          settings:
-            index:
-              mapping.source.mode: stored
+use no-op _source.mode attr:
+  - requires:
+      test_runner_features: [ "warnings" ]
 
   - do:
-      indices.put_mapping:
-        index: test_modify_source_mode_stored_disabled
-        body:
-          _source:
-            mode: disabled
-  - is_true: acknowledged
-
-  - do:
-      indices.get_mapping:
-        index: test_modify_source_mode_stored_disabled
-  - match: { test_modify_source_mode_stored_disabled.mappings._source.mode: stored }
-
----
-modify source mapping from stored to synthetic after index creation:
-  - do:
+      warnings:
+        - "Configuring source mode in mappings is deprecated and will be removed in future versions. Use [index.mapping.source.mode] index setting instead."
```

Comment:
this diff is messy, but just tests that _source.mode mapping attribute  is allowed to be used but is snoop and emits deprecation warning.